maintainers/maintainer-list.nix

```diff
@@ -8285,6 +8285,12 @@
     githubId = 1973389;
     name = "Reuben D'Netto";
   };
+  reardencode = {
```

Comment:
changes to maintainer-list.nix should be separate commits

Reply:
Done, good catch.